lib/chef/knife/bootstrap/windows-chef-client-msi.erb

```diff
@@ -107,12 +107,15 @@ goto install
 :install
 @rem Install Chef using chef-client MSI installer
 
-<% url="https://www.chef.io/chef/download?p=windows&pv=%MACHINE_OS%&m=%MACHINE_ARCH%" -%>
-<% url += latest_current_windows_chef_version_query -%>
-@set "REMOTE_SOURCE_MSI_URL=<%= url %>"
+<% default_url = "https://www.chef.io/chef/download?p=windows&pv=%MACHINE_OS%&m=%MACHINE_ARCH%" -%>
+<% default_url += latest_current_windows_chef_version_query -%>
+<% cs_url = msi_url.empty? ? default_url : msi_url -%>
+<% ps_url = msi_url.empty? ? default_url + "&DownloadContext=PowerShell" : msi_url -%>
+
+@set "REMOTE_SOURCE_MSI_URL=<%= cs_url %>"
+@set "REMOTE_SOURCE_MSI_PS_URL=<%= ps_url %>"
```

Comment:
Hmmm...  well currently, if they depended on it, the log would show the wrong thing because &DownloadContext is not added to REMOTE_SOURCE_MSI_URL - it's just being dumped into the url during the ps invocation below.  I'll try to refactor this so that REMOTE_SOURCE_MSI_URL is being set to the right thing after we decide if we are using PS or not.